next/backend/utils/fetchers/pagesFetcher.ts

```diff
@@ -0,0 +1,25 @@
+import { PageMeili } from '@utils/meiliTypes'
```

Comment:
Do you think it would be okay to refactor this right after we make it work? I think it iwill be natural step because we also want to reuse in on City Library project.